pandas/io/sas/sas.pyx

```diff
@@ -1,13 +1,19 @@
 # cython: profile=False
 # cython: boundscheck=False, initializedcheck=False
 from cython cimport Py_ssize_t
+from libc.stdint cimport (
+    int64_t,
+    uint8_t,
+    uint16_t,
+    uint32_t,
+    uint64_t,
```

Comment:
just curious, are these interchangeable with the versions of these we cimport from numpy?

Reply:
Interesting, I didn't know those exist.